child_compassion/views/child_compassion_view.xml

```diff
@@ -41,6 +41,9 @@
                             <button name="get_infos" string="get info" type="object" icon="fa-refresh" class="oe_stat_button"
                                     context="{'async_mode': False}"/>
                         </div>
+                        <div class="oe_right oe_button_box" name="buttons2">
+                            <button name="get_lifecycle_event" string="get lifecycle" type="object" icon="fa-refresh" class="oe_stat_button" context="{'async_mode': False}"/>

```

Comment:
Instead of putting a new div, you can place the button just after the previous one in the previous div